discretisedfield/region.py

```diff
@@ -136,9 +143,9 @@ def pmin(self):
 
         Returns
         -------
-        numpy.ndarray (3,)
+        numpy.ndarray
 
-            Point with minimum coordinates :math:`(p_x^\text{min},
+            Point with minimum coordinates for tree dimensions :math:`(p_x^\text{min},
```

Comment:
```suggestion
            Point with minimum coordinates. E.g. for three dimensions :math:`(p_x^\text{min},
```

Reply:
@swapneelap does this not give an array of `len(ndims)`?

Reply:
@samjrholt Sorry I did not get your point.